hkmc2/shared/src/main/scala/hkmc2/codegen/Lowering.scala

```diff
@@ -11,7 +11,7 @@ import hkmc2.semantics.{Term => st}
 
 import syntax.{Literal, Tree}
 import semantics.*
-import semantics.Term.*
+import semantics.Term.{Throw => _, *}
```

Comment:
Oh I made this change because of the name clash between `Term.Throw` and `codegen.Throw`, and `Term.Throw` is only used [here](https://github.com/CrescentonC/mlscript/blob/dbf5b01aa5c6191373bd20b94bd70e81f9abb1e6/hkmc2/shared/src/main/scala/hkmc2/codegen/Lowering.scala#L58) by `st.Throw`, while `codegen.Throw` is already used [here](https://github.com/CrescentonC/mlscript/blob/dbf5b01aa5c6191373bd20b94bd70e81f9abb1e6/hkmc2/shared/src/main/scala/hkmc2/codegen/Lowering.scala#L264), so I hide the one imported from `Term`.